### PR TITLE
fix(dashboards): Hide Retention line graph in insight cards

### LINKED ISSUE #9099: Retention dashboard panel shows line graph in background
## Bug description

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  

Screenshot 2022-03-18 at 07 59 29

## Expected behavior

Just show one or the other - table or line graph

## How to reproduce

1. Add a retention panel to a dashboard
2.
3.

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

Closes #9099.
This is a [Quality Launch Week](https://github.com/PostHog/posthog/projects/15) problem.

## Changes

All in the title and the linked issue.

### CODE REVIEW

File: frontend/src/scenes/retention/RetentionContainer.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function RetentionContainer({
 }): JSX.Element {
     return (
         <div className="retention-container">
-            <RetentionLineGraph inCardView={inCardView} inSharedMode={inSharedMode} />
+            {!inCardView && <RetentionLineGraph inCardView={inCardView} inSharedMode={inSharedMode} />}
```
Comment (pauldambra) [inline frontend/src/scenes/retention/RetentionContainer.tsx:14]: looks like we can remove the `inCardView` prop entirely now?

**Review by pauldambra — CHANGES_REQUESTED** [review]
there's even more deletion that can be done

File: frontend/src/scenes/retention/RetentionContainer.tsx (hunk 1 above)
Comment (Twixes) [reply frontend/src/scenes/retention/RetentionContainer.tsx]: Ofc

Comment (pauldambra) [reply frontend/src/scenes/retention/RetentionContainer.tsx]: and here toohttps://github.com/PostHog/posthog/blob/12a69c1965343dbfc0a932ee682d4672c11cf515/frontend/src/scenes/retention/RetentionLineGraph.tsx#L14

?? ☺️

**Review by pauldambra — APPROVED** [review]
I think there's one more inCardView deletion that can happen but this changes is approvable with or without that (should delete it if it can be though!)

**Pull request merged**